app/services/openlibrary_service.rb

```diff
@@ -0,0 +1,17 @@
+class OpenLibraryService
+  include HTTParty
+  base_uri 'openlibrary.org'
+  def initialize
```

Comment:
There is no need to do this in the initialize

Reply:
Done!